troubleshooting.md

```diff
@@ -0,0 +1,570 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Bootstrap VM issues](#bootstrap-vm-related-issues)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Node Related Issues](#node-related-issues)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+**Pre-requisites:** 
+
+First and foremost the BM installations follows a general architecture:
+* provisioner node 
+* master node 
+* worker node
+
+Before running the installation, please make sure you have a cleaned the previous files/manifests.
+
+# Bootstrap VM related issues 
+
+* Cannot login to the Bootstrap VM
+```
+$ ssh core@172.22.0.2
+Permission denied (publickey,password,keyboard-interactive).
+```
+
+Generally if the SSH keys are not updated in `install-config.yaml` file you cannot login to the bootstrap VM, add/update the correct SSH keys. 
+
+Make sure this file has the SSH keys of the provisioner node, if not you can't login into any of the nodes after deployment is completed.
```

Comment:
my comment above takes care of this sentence too, so feel free to remove.

Reply:
Removed this as suggested.